src/olympia/reviewers/tests/test_forms.py

```diff
@@ -151,6 +151,25 @@ def test_versions_queryset(self):
         assert list(form.fields['versions'].queryset) == [
             self.addon.current_version]
 
+    def test_versions_queryset_contains_pending_version(self):
+        addon_factory()
+        file_factory(version=self.addon.current_version,
+                     status=amo.STATUS_AWAITING_REVIEW)
```

Comment:
this adds an extra file to `current_version` (which would contain a single file with status `STATUS_PUBLIC` already) so you're not changing the number of versions with valid files shown.